ux.symfony.com/src/LiveMemory/Asset/CssCompiler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\LiveMemory\Asset;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\Compiler\AssetCompilerInterface;
+use Symfony\Component\AssetMapper\MappedAsset;
+use Symfony\Component\Filesystem\Path;
+
+class CssCompiler implements AssetCompilerInterface
```

Comment:
Could you just add a short description of the purpose of the compiler?

Reply:
Done!